plugins/dashboards/frontend/public/javascripts/countly.helpers.js

```diff
@@ -134,40 +134,39 @@
             },
             types: function() {
                 var self = this;
-                var result = this.allTypes.filter(function(item) {
+                if (this.enabledTypes && !this.enabledTypes.length) {
```

Comment:
fixes a small bug for data type that was not showing any data type options when enabledTypes was empty.